modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -36,6 +36,8 @@ def download_and_process
         response = submit(metadata, parts)
         process_response(response)
         log_submission(@upload, metadata)
+      rescue Faraday::TimeoutError
+        VBADocuments::UploadSubmission.refresh_statuses!([@upload])
```

Comment:
Per our conversation on zoom, if you want to look into this actually being called that may be something to look into. Otherwise the PR looks good to me.